roles/hotfix/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# Name of the container image to be hotfixed
+container_image_name: ""
```

Comment:
We have copied the hotfix code from tripleo-common. I have tried this interface. I think we can follow the tripleo docs: https://docs.openstack.org/project-deploy-guide/tripleo-docs/latest/deployment/container_image_prepare.html#building-hotfixed-containers and check what interface of hotfix works and add a example in the tcib. It will help to create the new role.

Reply:
Should we close this pr and have new one for adding example in the tcib?

Reply:
i am closing this pr since we no longer require this